AnotherSmallFile.txt

```diff
@@ -1,117 +0,0 @@
-Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-Phasellus porta dui non massa pellentesque semper.
-Nunc eget maximus metus.
-Donec laoreet aliquam leo ac tempus.
-Donec nulla ipsum, ultricies ac elit vel, dapibus fringilla neque.
-Proin sit amet consectetur velit.
-Morbi facilisis purus nec pretium mattis.
-Phasellus eu augue commodo, tincidunt nisl quis, porta nisi.
-
-Ut blandit vitae ligula ut auctor.
-Pellentesque nisl lectus, sagittis non semper quis, posuere quis tortor.
-Nullam bibendum aliquam luctus.
-Suspendisse pretium, magna sed dictum feugiat, augue lacus congue turpis, sit amet egestas ex nisi at leo.
```

Comment:
Reply from CodeFlow.

---
In reply to: [68509867](https://github.com/kivancmuslu/GitHubTest/pull/1#discussion_r68509867) [](ancestors = 68509867)